### PR TITLE
Add flake8-blind-except

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/flake8-blind-except/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (scopatz) [inline recipes/flake8-blind-except/meta.yaml]: is setuptools a runtime dependency?

Comment (sannykr) [reply recipes/flake8-blind-except/meta.yaml]: https://github.com/elijahandrews/flake8-blind-except/blob/master/setup.py
Line 25.

Comment (sannykr) [reply recipes/flake8-blind-except/meta.yaml]: If it is not a run requirement, please let me know and I will remove it from the list of run requirement.

Comment (scopatz) [reply recipes/flake8-blind-except/meta.yaml]: It does not seem to be a direct runtime depenendcy, looking at the code https://github.com/elijahandrews/flake8-blind-except/blob/master/flake8_blind_except.py

Comment (sannykr) [reply recipes/flake8-blind-except/meta.yaml]: Thanks. I will remove setuptools from the list of run dependencies.

**Comment by sannykr** [comment]
@scopatz : Could you please review and merge.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**